apps/frontend/src/hooks/useWalletConnect.ts

```diff
@@ -0,0 +1,27 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { WalletState } from '@sovryn/onboard-core';
+
+import { onboard } from '../lib/connector';
+
+export const useWalletConnect = () => {
+  const [wallets, setWallets] = useState<WalletState[]>([]);
+
+  const connectWallet = useCallback(() => {
+    onboard.connectWallet('injected');
+  }, []);
```

Comment:
Allow user to select wallet type from list (btw looks like `injected` doesn't really work, correct would be `Metamask`, `Brave` and etc.)
```suggestion
  const connectWallet = useCallback(() => {
    onboard.connectWallet();
  }, [])
```